lib/utils/DevServerPlugin.js

```diff
@@ -35,36 +35,44 @@ class DevServerPlugin {
     // TODO show warning about this
     const isSockJSType = options.webSocketServer.type === 'sockjs';
 
-    /** @type {string} */
-    let host = '';
+    let webSocketURL = '';
+
+    if (typeof options.client.webSocketURL === 'string') {
+      webSocketURL = options.client.webSocketURL;
```

Comment:
So user input should be like this in case of string `&host=0.0.0.0&port=8080&path=foo`?

Reply:
Yeah I think that's the only possible way since it's resolving a client path before it, not very convenient for user I agree

/cc @alexander-akait should confirm

Reply:
It doesn't make sense actually, most likely user will pass an absolute URL and we can use it directly

Reply:
It should work like `public`. And then we remove options.public in favor of `webSocketURL`

Reply:
> It should work like public. And then we remove options.public in favor of webSocketURL

Yes, right direction

Reply:
yep lets update